src/full/Agda/Syntax/Concrete/Operators.hs

```diff
@@ -544,8 +547,9 @@ parsePat prs = \case
     p@IdentP{}       -> return p
     RecP r fs        -> RecP r <$> mapM (traverse (parsePat prs)) fs
     p@EqualP{}       -> return p -- Andrea: cargo culted from DotP
-    EllipsisP _      -> fail "bad ellipsis"
     WithP r p        -> WithP r <$> parsePat prs p
+    p@(EllipsisP r Nothing) -> return p
+    EllipsisP r (Just p)    -> EllipsisP r . Just <$> parsePat prs p
```

Comment:
I would suppose this properly parses inside a `EllipsisP`, but somehow operators are not parsed correctly (at least if involving the `top` operator, i.e., the name of the function whose lhs I am parsing).